cmd/kubeadm/app/constants/constants.go

```diff
@@ -500,6 +500,7 @@ var (
 		21: "3.4.13-0",
 		22: "3.5.1-0",
 		23: "3.5.1-0",
```

Comment:
What is the reason for leaving 1.22 and 1.23 at 3.5.1? This is a critical bug, so IMO, it would make sense to bump the version for older releases as well.

Reply:
kubeadm 1.24 for the majority of users is used to deploy a 1.24 k8s.
kubeadm also supports skew of N-1 against k8s, meaning it can deploy k8s 1.23.

the 1.24 kubeadm / 1.22 k8s skew is technically unsupported.
but seems fine to me to also update all 3.5.x versions to be at patch release 3.5.3.

```
22: "3.5.3-0",
23: "3.5.3-0",
24: "3.5.3-0",
```

as a side note i'm planning to backport the kubeadm etcd patch bumps due to the complains about the bugs.